packages/circle-demo-webapp/app/components/WalletSend/WalletSend.tsx

```diff
@@ -93,38 +105,40 @@ export function WalletSend({
       }, 1000);
     }
   };
+
   return (
     <div className="items-center w-full">
-      <Form
-        method="post"
-        className="w-full"
-        onSubmit={(e) => {
-          handleSubmit(e).catch(console.error);
-        }}
-      >
+      {/* eslint-disable-next-line @typescript-eslint/no-misused-promises */}
```

Comment:
what misused promise is here? this looks like a code smell, can we figure out what is wrong here instead of ignoring the issue?

Reply:
the form except not async function. so eslint produce error

Reply:
the form expect not async function. that's why eslint produce error